Lib/tkinter/__init__.py

```diff
@@ -1732,11 +1736,14 @@ def getint_event(s):
         if any(isinstance(s, tuple) for s in args):
             args = [s[0] if isinstance(s, tuple) and len(s) == 1 else s
                     for s in args]
-        nsign, b, f, h, k, s, t, w, x, y, A, E, K, N, W, T, X, Y, D = args
-        # Missing: (a, c, d, m, o, v, B, R)
+        nsign, b, d, f, h, k, s, t, w, x, y, A, E, K, N, W, T, X, Y, D = args
```

Comment:
Is it breaking change? Because `f` argument is moved one position to the right and number of arguments is changed. Just making a point in case you have missed it.